src/tests/data/stream/chacha.vec

```diff
@@ -1,3 +1,9 @@
+[ChaCha(8)]
```

Comment:
Tests for the `seek` function are also needed. Add support in the tests (src/tests/test_stream.cpp) for an Seek parameter (which defaults to 0 for compatibility with existing tests), then using another implementation create ChaCha test vectors using at least Seek 1...65 and then also very large offsets.